### ISSUE TITLE
Error: Can't find variable: location

### ISSUE THREAD

**Issue by pesterhazy** [decker405/figwheel-react-native#1]
Project looks great, thanks for working on this!

Following your steps, after starting I see "Hello figwheel", I get a bunch of "Evaled" messages in the Xcode log. Finally I see a red screen:

```
2015-10-22 00:05:32.774 [error][tid:com.facebook.React.JavaScript] 'Error: Can\'t find variable: location\n stack: \n  <unknown>       index.ios.bundle?…:53636\n  <unknown>       index.ios.bundle?…:6133\n  callTimer       index.ios.bundle?…:5917\n  callTimers      index.ios.bundle?…:5940\n  __callFunction  index.ios.bundle?…:5549\n  <unknown>       index.ios.bundle?…:5487\n  guard           index.ios.bundle?…:5442\n  <unknown>       index.ios.bundle?…:5487\n  <unknown>       index.ios.bundle?…:5484\n  perform         index.ios.bundle?…:7029\n  batchedUpdates  index.ios.bundle?…:16541\n  batchedUpdates  index.ios.bundle?…:6332\n  <unknown>       index.ios.bundle?…:5483\n  guard           index.ios.bundle?…:5442\n  processBatch    index.ios.bundle?…:5482\n URL: undefined\n line: 1534\n message: Can\'t find variable: location'
````

The react-native package shows that requests are made:

```
[12:08:30 AM] <START> request:/index.ios.bundle?platform=ios&dev=true
[12:08:30 AM] <END>   request:/index.ios.bundle?platform=ios&dev=true (143ms)
::1 - - [21/Oct/2015:22:08:30 +0000] "GET /flow/ HTTP/1.1" 404 18 "-" "figTest/1 CFNetwork/711.4.6 Darwin/14.1.0"
[12:08:35 AM] <START> request:/index.ios.map?platform=ios&dev=true
[12:08:35 AM] <END>   request:/index.ios.map?platform=ios&dev=true (133ms)
```

Any idea what could be wrong?

**Comment by mfikes** [comment]
FWIW, I got the same, but dismissed the error display and successfully moved on.

**Comment by decker405** [comment]
That one's a little weird. I can't seem to reproduce it.

Looks like it's during one of the evals though. It may possibly be goog.uri looking for the document.location variable? I'll look into it a bit later tonight, hopefully I can reproduce it.

**Comment by decker405** [comment]
@pesterhazy: I pushed up some changes. I'm now using the native importScripts available in web workers to load the javascript and have added a try/catch around the import and eval. I've also excluded cljsjs/react so that you no longer have to clear out the react.inc.js file. 

Still haven't seen that problem though; but let me know if it still comes up. Hopefully the errors will have a bit more information this time.
